src/OrbitQt/CallTreeViewItemModelTest.cpp

```diff
@@ -19,6 +19,7 @@
 #include "QtUtils/AssertNoQtLogWarnings.h"
 
 constexpr uint64_t kCallstackId = 1;
+constexpr uint64_t kCallstackErrorId = 2;
```

Comment:
I would probably call this `kUnwind(ing)ErrorCallstackId`. This sounds like this is an "error id".